### PR TITLE
Reorg and updates

### PR DESCRIPTION
**Pull request by reedan88**
I reorganized my OOINet M2M tool and I also added the WFP Salinity Drift correction notebook here.

### CODE REVIEW

File: discrete_summary_checker/example/Check_Format.py [hunk 1]
```diff
@@ -167,17 +167,17 @@ def is_same(series):
     Column('CTD Conductivity 2 Flag', [MatchesPatternValidation(r"\*0|1{16}") | MatchesPatternValidation("-9999999")]),
     
     # Practical salinity should be within ocean ranges (32, 37) and floats
```
Comment (snwhite) [inline discrete_summary_checker/example/Check_Format.py]: The comment text says the range should be 32 to 37, but the checked range is now 31 to 37.

File: discrete_summary_checker/example/Check_Format.py [hunk 2]
```diff
@@ -206,7 +206,7 @@ def is_same(series):
     #     Phosphate: Maximum value ~5 uM (WOA 2018 mean fields); check for "<" which means undetecable
     Column('Discrete Phosphate [uM]', [InRangeValidation(0, 5) | MatchesPatternValidation(r"<\d.\d{2}") | MatchesPatternValidation("-9999999")]),
     #     Silicate: Maximum value for Southern Ocean ~120 uM (WOA 2018 mean fields); check for "<" which mean undetectable
```
Comment (snwhite) [inline discrete_summary_checker/example/Check_Format.py]: Change the comment to 150 to match the updated checked value.

File: discrete_summary_checker/example/Check_Format.py [hunk 3]
```diff
@@ -217,22 +217,22 @@ def is_same(series):
     Column('Discrete Nutrients Replicate Flag', [MatchesPatternValidation(r"\*0|1{16}") | MatchesPatternValidation("-9999999")]),
     
     # Salinity: Check that the ranges are within physical ocean ranges
-    Column('Discrete Salinity [psu]', [InRangeValidation(33, 37) | MatchesPatternValidation("-9999999")]),
+    Column('Discrete Salinity [psu]', [InRangeValidation(32, 37) | MatchesPatternValidation("-9999999")]),
     Column('Discrete Salinity Flag', [MatchesPatternValidation(r"\*0|1{16}") | MatchesPatternValidation("-9999999")]),
     Column('Discrete Salinity Replicate Flag', [MatchesPatternValidation(r"\*0|1{16}") | MatchesPatternValidation("-9999999")]),
     
     # Carbon System measurement: Check within ocean ranges; don't collect/measure pCO2
     #     Alkalinity: Should be between 2200 - 2400
```
Comment (snwhite) [inline discrete_summary_checker/example/Check_Format.py]: Update the comment value (2200) to match the checked value (2100).

File: discrete_summary_checker/example/Check_Format.py [hunk 4]
```diff
@@ -217,22 +217,22 @@ def is_same(series):
...
-    Column('Discrete Alkalinity [umol/kg]', [InRangeValidation(2200, 2400) | MatchesPatternValidation("-9999999")]),
+    Column('Discrete Alkalinity [umol/kg]', [InRangeValidation(2100, 2400) | MatchesPatternValidation("-9999999")]),
     Column('Discrete Alkalinity Flag', [MatchesPatternValidation(r"\*0|1{16}") | MatchesPatternValidation("-9999999")]),
     Column('Discrete Alkalinity Replicate Flag', [MatchesPatternValidation(r"\*0|1{16}") | MatchesPatternValidation("-9999999")]),
     #     DIC: Range should be 1900 - 2300
-    Column('Discrete DIC [umol/kg]', [InRangeValidation(1900, 2300) | MatchesPatternValidation("-9999999")]),
+    Column('Discrete DIC [umol/kg]', [InRangeValidation(1900, 2450) | MatchesPatternValidation("-9999999")]),
```
Comment (snwhite) [inline discrete_summary_checker/example/Check_Format.py:230]: Update the comment value (2300) to match the checked value (2450).

**Review by snwhite — COMMENTED** [review]
@reedan88 I have a few minor suggestions on updates to some of the comment text.  Also, pretty much every file has a whitespace change.  Just curious if they are line ending fixes/corrections?

**Comment by snwhite** [comment]
@reedan88 I think the Writeup.docx draft should be removed.

File: discrete_summary_checker/example/Check_Format.py (hunk 1 above)
Comment (reedan88) [reply discrete_summary_checker/example/Check_Format.py]: Yeah, a lot of these were set based on Irminger and then I started applying them elsewhere and it was throwing a lot of errors for like Pioneer and I loosened the checked range.

**Comment by reedan88** [comment]
@snwhite Removed the Writeup.docx and pushed an updated version with the data citation suggested by Stace and with the figures rendered.

**Review by snwhite — DISMISSED** [review]

File: discrete_summary_checker/example/Check_Format.py [hunk 5]
```diff
@@ -166,20 +166,20 @@ def is_same(series):
...
+    Column('CTD Oxygen Saturation [mL/L]', [DecimalValidation, InRangeValidation(4,9) | MatchesPatternValidation("-9999999")]),
     
     # Fluorescence - we don't measure this, should be fill value -9999999
-    Column('CTD Fluorescence [mg/m^3]', [MatchesPatternValidation("-9999999")]),
-    Column('CTD Fluorescence Flag', [MatchesPatternValidation("-9999999")]),
+    Column('CTD Fluorescence [mg/m^3]', [DecimalValidation | MatchesPatternValidation("-9999999")]),
+    Column('CTD Fluorescence Flag', [MatchesPatternValidation(r"\*0|1{16}") | MatchesPatternValidation("-9999999")]),
     
-    # Beam Attenuation (0, 1) and Transmission (0, 100)
+    # Beam Attenuation (-0.01, 1) and Transmission (-1, 101)
```
Comment (rtravis-whoi) [inline discrete_summary_checker/example/Check_Format.py]: Beam Attenuation is (-0.01, 1) on line 182 but is (-0.1,1) below on line 183 - should be consistent one way or the other.

**Review by rtravis-whoi — CHANGES_REQUESTED** [review]
I reviewed the discrete_summary_checker/example/Check_Format.py file ONLY.  Requesting one change: Beam Attenuation is (-0.01, 1) on line 182 but is (-0.1,1) below on line 183 - should be consistent one way or the other.

File: discrete_summary_checker/example/Check_Format.py [hunk 6]
```diff
@@ -33,7 +33,7 @@
 # #### Summary Sheet
 # Load the summary sheet. Make sure to navigate to the correct directory and have the correct file name entered.
 
-summary_sheet = pd.read_excel("../data/Irminger_Sea-06_AR35-05_Discrete_Sample_Summary.xlsx")
+summary_sheet = pd.read_excel("/home/andrew/Documents/OOI-CGSN/QAQC_Sandbox/Ship_data/data/Papa-04/Station_Papa-04_RB1605_Discrete_Summary_2022-01-03_ACR.xlsx")
```
Comment (PaulWhelan-whoi) [inline discrete_summary_checker/example/Check_Format.py]: Is this a path that is going to be accessible? Does the file need to be moved to ../data or some other accessible location?

File: discrete_summary_checker/example/Check_Format.py [hunk 7]
```diff
@@ -166,20 +166,20 @@ def is_same(series):
...
+    Column('CTD Oxygen Saturation [mL/L]', [DecimalValidation, InRangeValidation(4,9) | MatchesPatternValidation("-9999999")]),
     
     # Fluorescence - we don't measure this, should be fill value -9999999
-    Column('CTD Fluorescence [mg/m^3]', [MatchesPatternValidation("-9999999")]),
-    Column('CTD Fluorescence Flag', [MatchesPatternValidation("-9999999")]),
+    Column('CTD Fluorescence [mg/m^3]', [DecimalValidation | MatchesPatternValidation("-9999999")]),
+    Column('CTD Fluorescence Flag', [MatchesPatternValidation(r"\*0|1{16}") | MatchesPatternValidation("-9999999")]),
     
-    # Beam Attenuation (0, 1) and Transmission (0, 100)
+    # Beam Attenuation (-0.1, 1) and Transmission (-1, 101)
```
Comment (rtravis-whoi) [inline discrete_summary_checker/example/Check_Format.py:182]: correction looks good

File: discrete_summary_checker/example/Check_Format.py [hunk 8]
```diff
@@ -293,3 +296,7 @@ def is_same(series):
     merrors = metadata_schema.validate(station_data)
     for error in merrors:
         print(error)
+
+
+
+
```
Comment (rtravis-whoi) [inline discrete_summary_checker/example/Check_Format.py:302]: update to line 182 looks good

**Review by rtravis-whoi — DISMISSED** [review]

File: discrete_summary_checker/example/Check_Format.py (hunk 6 above)
Comment (reedan88) [reply discrete_summary_checker/example/Check_Format.py]: I updated it to a generic placeholder of "put in path to sample sheet"

**Review by PaulWhelan-whoi — APPROVED** [review]

**Review by snwhite — APPROVED** [review]

**Pull request merged**